doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,33 @@
+# QoS policies
+
+Unlike other middlewares, e.g. [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html),
+iceoryx keeps things simple and offers basic Quality of Service (QoS) settings. The QoS
+settings are called options and can be used to optimize and tailor the communication.
+
+## `PublisherOptions` and `SubscriberOptions`
+
+The three most important settings are:
+
+| Option                              | Explanation                                                                                                                                       |
+|-------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| `PublisherOptions::historyCapacity` | This enables late-joining subscribers to request the n last samples                                                                               |
+| `SubscriberOptions::queueCapacity`  | Size of the subscriber queue where samples are stored before they are passed to the user                                                          |
+| `SubscriberOptions::historyRequest` | The number of samples a late-joining subscriber will request from a publisher, should be equal or smaller than `historyCapacity` otherwise capped |
+
+For more information about the options see the corresponding example [`iceoptions`](iceoptions.md).
+
+## Publisher subscriber matching criteria
+
+Two criteria have to be fulfilled in order for a publisher and a subscriber to be connected.
+
+ 1. Same `capro::ServiceDescription`
+ 2. Matching `SubscriberTooSlowPolicy` and `QueueFullPolicy` in `PublisherOptions` and `SubscriberOptions`
+
```

Comment:
@mossmaurice there is a third criteria but I forgot the exact semantics. I think @MatthiasKillat can tell you about it

Reply:
Added.